bin/check-s3-bucket-visibility.rb

```diff
@@ -0,0 +1,131 @@
+#! /usr/bin/env ruby
+#
+# check-s3-bucket-visibility
+#
+# DESCRIPTION:
+#   This plugin checks a bucket for website configuration and bucket policy.
+#   It alerts if the bucket has a website configuration, or a policy that has
+#   Get or List actions.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#
+# USAGE:
+#   ./check-s3-bucket-visibility.rb --bucket-name mybucket --aws-region eu-west-1
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright (c) 2015, Olivier Bazoud and Ricky Hussmann,
+#     olivier.bazoud@gmail.com, ricky.hussmann@gmail.com
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckS3Bucket < Sensu::Plugin::Check::CLI
+  option :aws_access_key,
+         short: '-a AWS_ACCESS_KEY',
+         long: '--aws-access-key AWS_ACCESS_KEY',
+         description: "AWS Access Key. Either set ENV['AWS_ACCESS_KEY'] or provide it as an option",
+         default: ENV['AWS_ACCESS_KEY']
+
+  option :aws_secret_access_key,
+         short: '-k AWS_SECRET_KEY',
+         long: '--aws-secret-access-key AWS_SECRET_KEY',
+         description: "AWS Secret Access Key. Either set ENV['AWS_SECRET_KEY'] or provide it as an option",
+         default: ENV['AWS_SECRET_KEY']
+
+  option :aws_region,
+         short: '-r AWS_REGION',
+         long: '--aws-region REGION',
+         description: 'AWS Region (defaults to us-east-1).',
+         default: 'us-east-1'
+
+  option :aws_session_token,
+         short: '-t AWS_SESSION_TOKEN',
+         long: '--aws-session-token TOKEN',
+         description: 'AWS Session Token',
+         default: ENV['AWS_SESSION_TOKEN']
+
+  option :bucket_name,
```

Comment:
might want to include multiple bucket names to avoid needing instantiate multiple checks for multiple buckets. Something like a comma separated list of buckets seems appropriate. See #24 , #27  for more details as to why.